src/controllers/footage.controller.ts

```diff
@@ -158,10 +170,74 @@ const getFootageClips = async (
   return res.status(200).json({ data: footageClips });
 };
 
-// TODO: Implement updateFootage endpoint to update after parsing & analysis.
-// const updateFootage = async (req: Request, res: Response) => {
-//
-// };
+/**
+ * PATCH /footage/:uuid
+ * @summary Endpoint to update a specific Footage document.
+ * @return {string, FootageDocument} 200 - Success response returns the Footage document updated and a message.
+ * @return 400 - Fields isCsgoFootage & isAnalyzed were not provided.
+ * @return 406 - One of the params (UUID, isCsgoFootage or isAnalyzed) was not provided.
+ * @return 412 - No document with the provided UUID was found.
+ * @return 418 - An error occured while attempting to update the FootageDocument.
+ */
```

Comment:
this is a really good idea, and something we should work on implementing as we make more changes

Reply:
sounds good, I saw b3kn implementing it and used it for the endpoints I coded.